diesel/src/pg/backend.rs

```diff
@@ -13,6 +13,11 @@ use sql_types::{Oid, TypeMetadata};
 #[derive(Debug, Clone, Copy, Hash, PartialEq, Eq)]
 pub struct Pg;
 
+pub struct PgValue {
+    pub data: Vec<u8>,
```

Comment:
I'd like all these fields to be private. The struct should not be destructurable

Reply:
Fixed, added new method.